jet-fuel/config/boot.rb

```diff
@@ -0,0 +1,3 @@
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
+
+require 'bundler/setup' # Set up gems listed in the Gemfile.
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.